jobs/utils.py

```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
```

Comment:
Should we rename this file to 'job(s)_utils.py'? So that we do not need to rename it when importing? I know it is a bit redundant but from the import PoV, it makes sense.

Reply:
Done.